CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 9.0.0 - 2025-05-29
```

Comment:
Q: should this be `major` bump?

Reply:
There is a breaking change that is not yet released. That is why the version was not bumped in this PR